src/vdf_io/turbopuffer.py

```diff
@@ -0,0 +1,95 @@
+import argparse
```

Comment:
this file needs to be put into `src/vdf_io/export_vdf/` and `src/vdf_io/import_vdf/` and needs to be called turbopuffer_export.py turbopuffer_import.py respectively.

the make_parser, import_vdb and export_vdb functions need to be member functions of ImportTurbopuffer and ExportTurbopuffer classes respectively.